.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,12 @@
 
 ## What is the purpose of the change
 
-XXXXX
+<!--
+If this PR fixes a GitHub issue, please add `fixes #<XXX>` or `closes #<XXX>`. Please refer to the documentation for more information:
+https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
+-->
+
+fix #<xxx> <!-- <xxx> replace with PR id -->
```

Comment:
![image](https://user-images.githubusercontent.com/21963954/218623666-388b89a0-672c-42a0-b121-d8d0ff66229b.png)
When I preview, there seems to be no text display

Reply:
@RongtongJin 

Your page is a completed page and this modification is used when creating a PR upon submission, as shown in the following figure.
![image](https://user-images.githubusercontent.com/15797831/218624875-14be6bd2-aba9-489a-9d3d-ad22c2bb582d.png)